### PR TITLE
workflow: add flow-latency workflow

### PR DESCRIPTION
**Pull request by masco**

### CODE REVIEW

**Comment by masco-enovance** [comment]
run skydive-functional-tests-backend-orientdb

File: statics/workflows/flow-latency.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        var packetInjection = new PacketInjection();
+        packetInjection.Src = "G.V().Has('TID', '" + from + "')"
+        packetInjection.Dst = "G.V().Has('TID', '" + to + "')"
+        packetInjection.Type = "icmp4"
+        packetInjection.ICMPID = Math.floor(Math.random() * 1000);
+        packetInjection.Count = 1
+
+        return client.captures.create(capture).then(function (c) {
+            capture = c
+            return sleep(2000)
```
Comment (nplanel) [inline statics/workflows/flow-latency.yaml]: may sleep 1 sec only, as it's seems to be enough to setup/start the capture

File: statics/workflows/flow-latency.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        }).then(function (nodes) {
+            if (nodes[0].Metadata.Neutron && nodes[0].Metadata.Neutron.IPV4) {
+                packetInjection.DstIP = nodes[0].Metadata.Neutron.IPV4[0]
+            }
+            if (nodes[0].Metadata.ExtID && nodes[0].Metadata.ExtID["attached-mac"]) {
+                packetInjection.DstMAC = nodes[0].Metadata.ExtID["attached-mac"]
+            }
+            return client.packetInjections.create(packetInjection)
+        }).then(function () {
+            return sleep(2000)
```
Comment (nplanel) [inline statics/workflows/flow-latency.yaml]: same here 1 sec is more than enough to packets

File: statics/workflows/flow-latency.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        }).then(function () {
+            return client.G.Flows().Has("ICMP.ID", packetInjection.ICMPID)
```
Comment (nplanel) [inline statics/workflows/flow-latency.yaml]: you should filter on TIDs too : like G.V().Has('TID', from).Flows("ICMP.ID", <id>)
As other captures and packets could be saved in Skydive

File: statics/workflows/flow-latency.yaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+                    "Latency(RTT)": (flows[0].RTT / 1000000).toFixed(3) + " ms"
+                }
+            } else {
+                return {
+                  "Error": "No flows found, First check the conectivity."
+                }
+            }
+        }).finally(function () {
+            client.captures.delete(capture.UUID)
+        })
```
Comment (nplanel) [inline statics/workflows/flow-latency.yaml]: please add .catch(
            client.captures.delete(capture.UUID)
)

**Comment by masco-enovance** [comment]
run skydive-cdd-overview-tests
run skydive-functional-tests-backend-orientdb

**Comment by masco-enovance** [comment]
run skydive-k8s-tests

**Pull request closed without merging**